docs/getting-started/provider-aws.md

```diff
@@ -0,0 +1,219 @@
+---
+title: AWS Quickstart
+weight: 2
+---
+
+Connect Crossplane to AWS to create and manage cloud resources from Kubernetes with the [Upbound AWS Provider](https://marketplace.upbound.io/providers/upbound/provider-aws).
+
+This guide walks you through the steps required to get started with the Upbound AWS Provider. This includes installing Crossplane, configuring the provider to authenticate to AWS and creating a _Managed Resource_ in AWS directly from your Kubernetes cluster.
+
+- [Prerequisites](#prerequisites)
+  - [Install the AWS provider](#install-the-aws-provider)
+  - [Create a Kubernetes secret for AWS](#create-a-kubernetes-secret-for-aws)
+    - [Generate an AWS key-pair file](#generate-an-aws-key-pair-file)
+    - [Create a Kubernetes secret with the AWS credentials](#create-a-kubernetes-secret-with-the-aws-credentials)
+  - [Create a ProviderConfig](#create-a-providerconfig)
+  - [Create a managed resource](#create-a-managed-resource)
+  - [Delete the managed resource](#delete-the-managed-resource)
+- [Next steps](#next-steps)
+
+## Prerequisites
+This quickstart requires:
+* a Kubernetes cluster with at least 3 GB of RAM
+* permissions to create pods and secrets in the Kubernetes cluster
+* [Helm] version `v3.2.0` or later
+* an AWS account with permissions to create an S3 storage bucket
+* AWS [access keys](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html#cli-configure-quickstart-creds)
+
+{{< hint type="tip" >}}
+If you don't have a Kubernetes cluster create one locally with [minikube](https://minikube.sigs.k8s.io/docs/start/) or [kind](https://kind.sigs.k8s.io/).
+{{< /hint >}}
+
+
+{{< hint type="note" >}}
+All commands use the current `kubeconfig` context and configuration. 
+{{< /hint >}}
+
+{{< include file="/docs/master/getting-started/install-crossplane.md" type="page" >}}
+
+### Install the AWS provider
```

Comment:
This shouldn't really be a subheading of prerequisites.

Reply:
Fixed in [4b0853a](https://github.com/crossplane/crossplane/pull/3396/commits/4b0853ac386d19f2944727f1d6b30d3924d80cd0)

Reply:
(feel free to mark this resolved)